### PR TITLE
Remove hard-coded credentials.

### PR DESCRIPTION
**Pull request by sprucely**
Replace with single SuperuserSettings from user secrets.
issue #247 

To pass super user into DbInitializer, I made SuperuserSettings inherit User rather than create a separate class and map it to User. I could have just used a IOptions<User>, but that would have felt dirty.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```
Comment (srkirkland) [inline package-lock.json]: @sprucely how did you get this file, there isn't even a package.json in this directory?  What is it supposed to do for us?

Comment (sprucely) [reply package-lock.json]: Wait, what? I thought I avoided staging any of the automatic package changes...

Comment (sprucely) [reply package-lock.json]: Oh, I reverted and force pushed in order to make the message more consistent with other commits, and those changes snuck back in.

Comment (sprucely) [reply package-lock.json]: Should I amend or commit another change to remove those?

Comment (srkirkland) [reply package-lock.json]: Just remove in another commit

File: src/Payments.Core/Helpers/DbInitializer.cs [hunk 2]
```diff
@@ -43,41 +47,10 @@ public async Task Initialize()
             var editorRole = await FindOrCreateRole(TeamRole.Codes.Editor);
             var reportRole = await FindOrCreateRole(TeamRole.Codes.ReportUser);
 
-            // create system users
-            if (!_context.Users.Any())
+            // create system user
+            if (!_context.Users.Any() && !string.IsNullOrWhiteSpace(_superuserSettings.CampusKerberos))
```
Comment (jSylvestre) [inline src/Payments.Core/Helpers/DbInitializer.cs]: @srkirkland @sprucely The logic hasn't changed, but with this code it never gets called in there are any users.
Maybe not how we want it to function.

Comment (srkirkland) [reply src/Payments.Core/Helpers/DbInitializer.cs]: Seems like a good backstop check to me.  Really I think this code's purpose was to get a new SQL express instances started up for dev/test, so only running when the db is empty seems like a smart safe choice.

**Pull request merged**